tests/plugins/pastefromword/generated/_helpers/createTestSuite.js

```diff
@@ -0,0 +1,73 @@
+/* global createTestCase */
+/* exported createTestSuite */
+
+// Creates a test suite based on the options provided. The test suite then should be passed to `bender.test` to run tests.
```

Comment:
Why not standard JSDoc comments, similar to our [existing tests](https://github.com/ckeditor/ckeditor-dev/blob/3ee603539f24cbebf0c0c24ec1b3eb48d678570b/tests/plugins/copyformatting/_helpers/tools.js#L64-L72)? Comments from tests don't leak into documentation, so it's safe to use JSDoc in a normal way.